Lib/ufoLib/__init__.py

```diff
@@ -121,11 +93,10 @@ class UFOReader(object):
 	"""Read the various components of the .ufo."""
 
 	def __init__(self, path):
-		if not os.path.exists(path):
-			raise UFOLibError("The specified UFO doesn't exist.")
-		self._path = path
+		self.fileSystem = FileSystem(path)
```

Comment:
yeah, that would be cool. Things would be much easier if we were to make pyfilesystem2 a required dependency, but I'm sure others would start complaining about dependency hell and stuff.